action/action-simulator/src/main/java/com/powsybl/action/simulator/loadflow/CaseExporter.java

```diff
@@ -36,27 +34,22 @@ public CaseExporter(Path outputCaseFolder, String outputCaseFormat) {
     }
 
     @Override
-    public void loadFlowDiverged(Network network, Contingency contingency, int round) {
-        exportNetwork(network, contingency, round);
+    public void loadFlowDiverged(Contingency contingency, Network network, int round) {
+        exportNetwork(contingency, network, round);
     }
 
     @Override
-    public void loadFlowConverged(Network network, Contingency contingency, int round, List<LimitViolation> violations) {
-        exportNetwork(network, contingency, round);
+    public void loadFlowConverged(Contingency contingency, List<LimitViolation> violations, Network network, int round) {
+        exportNetwork(contingency, network, round);
     }
 
-    private void exportNetwork(Network network, Contingency contingency, int round) {
+    private void exportNetwork(Contingency contingency, Network network, int round) {
         if (null != contingency && StringUtils.isNotEmpty(contingency.getId())) {
             String contingencyIdAndRound = contingency.getId().replace(' ', '_').replace('.', '_') + String.valueOf("_Round_" + round);
```

Comment:
I'm not sure it's a good ID to do replacement in the contingencyID: blank are valid in file names
The dots are also valid